pkgs/applications/misc/sioyek/default.nix

```diff
@@ -13,29 +12,21 @@
 , qt3d
 , qtbase
 , qmake
+, qtspeech
 , wrapQtAppsHook
 }:
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "sioyek";
-  version = "2.0.0";
+  version = "2.0.0-unstable-2024-01-25";
 
   src = fetchFromGitHub {
     owner = "ahrm";
     repo = "sioyek";
-    rev = "v${finalAttrs.version}";
-    sha256 = "sha256-GFZaTXJhoBB+rSe7Qk6H6FZJVXr3nO9XgM+LAbS4te4=";
+    rev = "36cff8dbec4bebdab441c21d4128e01fb89b4237";
```

Comment:
Is there a reason this specific commit is used rather than a more recent one?

Reply:
because this PR is like 5 months old and if i bump the version then i would need to test it again, and potentially address new issues. This version is still a whole lot better than what is currently in unstable.

Reply:
oh ok. Just fyi I've been using this [overlay](https://github.com/stephen-huan/nixos-config/blob/master/overlays/sioyek.nix) for the latest commit on the development branch and it seems to work

```nix
final: prev:

let
  self = prev.sioyek;
  sioyek' = self.overrideAttrs (previousAttrs: {
    src = final.fetchFromGitHub {
      owner = "ahrm";
      repo = "sioyek";
      rev = "b312fe0b7ae4b8125febef4467480367e0030665";
      sha256 = "sha256-0UCFp52mSEHFuH50FD+CrvmiR9SWfgqm8gUypX7gLG4=";
    };
    buildInputs = previousAttrs.buildInputs ++ [ final.qt6.qtspeech ];
    patches = [ ];
  });
in
{
  sioyek = final.qt6.callPackage sioyek'.override { };
}
```
(though have not tested extensively). I guess one of us can open a new pr once this one lands :p.

Reply:
Exactly my thought. I just really think this PR should land, its been too damn long. Don't want to mess it up.